src/app/quizPage.js

```diff
@@ -215,3 +217,45 @@ const resetQuizAfterQuestion = () => {
     quizBody.innerHTML = getTemplateContent(quizTemplate)[1].innerHTML // get the quiz body inner HTML from the template
 }
 
+
+const setupTimer = () => {
+    const barDiv = createTimer()
+    startTimer(barDiv);
+}
+
+const createTimer = () => {
+    const timerBody = document.getElementById('timer');
+    const bar = document.createElement("div");
+    bar.setAttribute('id', 'bar')
+    timerBody.appendChild(bar);
+
+    return bar
+}
+
+const startTimer = (bar) => {
+    var durationTime = 120; // czas w sekundach, można dowolnie zmianiać 120 -> 120 sekund = 2 minuty
+    printTime(durationTime);
+    bar.style.animation = "anim 1 linear forwards";
+    bar.style.animationDuration = durationTime+"s";
+    interval = setInterval(runningTime, 1000);
+    timeOut = setTimeout(function(){
+        clearInterval(interval);
+        bar.style.animationPlayState = "paused";
+    },(durationTime*1000));
+
+    function runningTime() {
+        if (durationTime) {
```

Comment:
`if (durationTime > 0)` would be better - what if somebody sets durationTime = 0.5 and then it will subtract 1 and it will become -0.5 (so `durationTime` would never be false/0)

Reply:
You are right. I corrected above mentioned condition -> if (durationTime > 0)